recipe/bld.bat

```diff
@@ -15,5 +15,5 @@ set "LDFLAGS="
 REM don't add d1trimfile option because clang doesn't recognize it.
 set "SRC_DIR="
 
-%PYTHON% -m pip install . -vv --no-deps --no-build-isolation
+%PYTHON% -m pip install . -vvv --no-deps --no-build-isolation
```

Comment:
```suggestion
%PYTHON% -m pip install . -vvv --no-deps --no-build-isolation --config-settings=compile-args="-j4"
```
Not tested, but this may help with making the windows build more stable.

Reply:
Added